src/pydocstyle/checker.py

```diff
@@ -122,6 +123,8 @@ class ConventionChecker:
         r"\s*"
         # Followed by a colon
         r":"
+        # Might have a new line
+        r"\n?",
```

Comment:
Did you test this? There shouldn't be a comma at the end of this line, it's not a tuple, it's a regular expression relying on string concatenation over multiple lines with comments to make it more readable.

Reply:
Yup, I was making some changes through the browser editor and didn't catch this. Will fix shortly.